src/main/resources/graphics/scenery/volumes/GridCellsToZero.comp

```diff
@@ -0,0 +1,27 @@
+#version 450
+#extension GL_EXT_debug_printf : enable
+#define USE_PRINTF 1
+
+layout (local_size_x = 16, local_size_y = 16) in;
+layout (set = 0, binding = 0, r32ui) uniform uimage3D GridCells;
+
+void main() {
```

Comment:
This could use some docs what the shader is doing

Reply:
c8af72639a008e849c57e95e025bc2f343c78b41 adds documentation